howtos/process/bug-tracking.rst

```diff
@@ -0,0 +1,234 @@
+  feature implementation or bug reproduction.
+
+* |label-need-info| - Further information is requested.
+
+.. |label-blocked| image:: images/label-blocked.png
+        :scale: 70
+
+.. |label-need-info| image:: images/label-need-info.png
+        :scale: 70
+
+How to Report a Bug
+********************
+Please
+`create a issue <https://help.github.com/articles/creating-an-issue/>`_
+and add label |label-bug|.
+
+And please provide the following information:
+
+* Title
+     * The title should be a clear and concise summary of the bug.
+
+     * The title must be unique and descriptive. Bad examples are
+       "ipc timeout" and "topology failed to load". Ideally the title
+       should contain keywords from the kernel, firmware, or user space
+       error message.
+
+     * The title should also contain a prefix indicating the area of
+       failure e.g. "ipc:", "topology:", "pipeline:"
+
+* Environment
+    * Branch name and commit hash of 3 repositories: sof (firmware),
+      linux (kernel driver) and soft (tools & topology).
+
+    * Name of the topology file
+
+    * Name of the platform(s) on which the bug is observed.
+
+    * Reproducibility Rate. If you can only reproduce it randomly,
+      it's useful to report how many times the bug has been reproduced
+      vs. the number of attempts it’s taken to reproduce the bug.
+
+* Steps to reproduce
+   * The steps must be precise. And please help to narrow down the steps.
+
+   * Please number the steps from beginning to end so developers can
+     easily follow through by repeating the same process
+
+* Expected Result
+   * Describe what the user should expect.
+
+* Actual Result
+   * In contrast to the expected behavior, describe what currently happens.
+
+* Proof
+    * Please paste the relevant dmesg and logger data to the comment box.
+      The pasted data should contain the actual crash or error but also
+      the conditions prior to the bug. i.e. also copy the 10 lines before
+      the crash.
+
+      For firmare boot failure, the pasted dmesg must include the
+      trace point which indicates the progress of firmware boot process:
+
+      |trace-point|
+
+    * Entire kernel message and firmware logger text should also be
+      attached for reference.
+
+    * If you cannot hear sound for playback or capture, please attach
+      your amixer settings. If there is a mixer setting seems wrong,
+      please paste the relevant amixer item in the comment box.
+
+    * For audio quality issues (eg. noise, glitch sound and distortion
+      etc), it's helpful to
+
+      * play/capture a sine wave, attach the captured wave file.
+      * share the parameters of the sine wave: frequency, sample rate,
+        format and number of channels.
+      * share the waveform snapshot where the glitch/distortion happens
+        by `Audacity <https://www.audacityteam.org/>`_.
+
+      This will help developers to debug possible DMA issues.
```

Comment:
Can you remove this line about DMA as it may mislead as all quality faults are not DMA related. I think it's probably best to show some examples of audacity captures here i.e. we need screenshot showing sine wave (say over 10ms) then another screenshot zooming in on a glitch(es) showing individual samples (where glitch starts and ends).

Reply:
Okay. Will do. Thanks for your review!

Reply:
@lgirdwood Updated in the v4 patch. Please check if the screenshots look good. Thanks!